app/views/handlers/block.py

```diff
@@ -116,3 +117,11 @@ def _set_started_at_metadata(self):
             started_at = datetime.utcnow().isoformat()
             logger.info("Survey started", started_at=started_at)
             collection_metadata["started_at"] = started_at
+
+    def _get_safe_page_title(self, title):
+        if title and "placeholders" in title:
+            if "text_plural" in title:
+                title = title["text_plural"]["forms"]["other"]
+            else:
+                title = title["text"]
```

Comment:
So there is a generator outside the class that already does this https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/questionnaire/questionnaire_schema.py#L412 but I am not overly comfortable with using it. It recursively looks for the key in sub dicts/list until it finds it, but if there was a duplication of the key in a sub dict it would also find it. There is an ignore list which helps but I think we need to be more explicit here

We would also have to map keys to different values, which isn't a problem but would need to be done i.e text plural = ["text_plural"]["forms"]["other"]

Reply:
Not a fan of the arg name either, we should try to not use 'block', considering its meaning in our schema

Reply:
I meant adding a method similar to https://github.com/ONSdigital/eq-translations/blob/master/eq_translations/survey_schema.py#L83.